testing/dead_field.js

```diff
@@ -0,0 +1,29 @@
+// Generated by js_of_ocaml
```

Comment:
What's this file ? Maybe committed by mistake ?

Reply:
compiler/tests-full/stdlib.cma.js was likely also committed by mistake

Reply:
Oops, yes I added those two accidentally in the most recent commit

Reply:
Should be fixed now.